targets/cloudwatch.libsonnet

```diff
@@ -20,6 +20,7 @@
     namespace,
     period            = 'auto',
     queryMode         = $.queryModes.Metrics,
+    logGroups         = null,
```

Comment:
why was this set to null?

Reply:
`null` is the default value. This is only needed for the linked PR, otherwise it can be null.